packages/asc-ui/src/components/Alert/Alert.stories.tsx

```diff
@@ -9,7 +9,7 @@ import Link from '../Link'
 
```

Comment:
Perhaps it's also a good idea to create a .mdx file from this, just have a look how I did that for the Checkbox component. Also, because the variants will be included in the Design System, I've created a story per variant for the checkbox, I think it's a good idea to also do that here!

Reply:
Fixed